testing/README.md

```diff
@@ -0,0 +1,31 @@
+# Testing
+
+Some resources to get us started:
+
+## RSpec
+
+* http://betterspecs.org/
+
+### RSpec Style Guide
+
+* https://github.com/reachlocal/rspec-style-guide
+
+## Code Coverage
+
+While imperfect, code coverage is a helpful tool to identify code that is never executed in the course of running tests.
+
+* put badge on repo README
+
+### Coveralls
+
+* works nicely with travis
+
+## Continuous Integration
+
+* whenever possible, CI results should be visible with the code repo.
+
+### Travis
+
+* use travis for CI for github repos
+* put travis badge on repo README
+* (which rubies to test)
```

Comment:
Should we mention anything about Jenkins on this page (perhaps in the context of internal services not available outside of the Stanford IP range)?